Makefile

```diff
@@ -30,7 +30,11 @@ default:
 .PHONY: test
 ## Run all tests and code reports
 test: .tox
-	tox -p auto
+	tox --develop -p auto
```

Comment:
I don't really know all the implications here, but on CI we'll run with `develop=False` as per `tox.ini` but this local version will not. That said, I usually use `py.test` or `trial` directly, so I'm even further from what CI does ;)

Reply:
Does [the commit message for that change](https://github.com/tahoe-lafs/tahoe-lafs/pull/830/commits/92de966c9169b4b68cf1868a2ceb9fcb95a90f3c) address the implications for you?

Reply:
May I resolve this one, @meejah ?